ERCS/erc-human-robot-societies.md

```diff
@@ -0,0 +1,510 @@
+---
```

Comment:
```suggestion
---
eip: 7777
```
Assigning next sequential EIP/ERC/RIP number.

Please also update the filename.